VitDeck/Assets/VitDeck/Validator/Rules/Booth/MissingFinder.cs

```diff
@@ -95,6 +103,39 @@ private void FindRecursive(Object unityObject)
             }
         }
 
+        private void CheckMaterial(Material material)
+        {
+            if (material.shader != null)
+            {
+                //設定中のシェーダーに存在するプロパティの取得
+                var shaderProperties = new HashSet<string>();
+                var count = ShaderUtil.GetPropertyCount(material.shader);
+                for (int i = 0; i < count; i++)
+                {
+                    string propName = ShaderUtil.GetPropertyName(material.shader, i);
+                    shaderProperties.Add(propName);
+                }
+
+                var matSo = new SerializedObject(material);
+                var matSp = matSo.FindProperty("m_SavedProperties");
+                var texEnvsSp = matSp.FindPropertyRelative("m_TexEnvs");
+                for (int i = 0; i < texEnvsSp.arraySize; i++)
+                {
+                    var prop = texEnvsSp.GetArrayElementAtIndex(i);
+                    var propName = prop.FindPropertyRelative("first").stringValue;
+                    //設定中のシェーダーに存在するテクスチャ指定プロパティのみmissing検証
+                    if (shaderProperties.Contains(propName))
+                    {
+                        var tex = prop.FindPropertyRelative("second.m_Texture");
+                        if (tex != null && IsMissing(tex))
+                        {
+                            missingProperties.Add(tex);
```

Comment:
以下のように変更することで、ログにプロパティ名が出るようになって何がmissingしているのか多少分かりやすくなると思いました。本当にmissingしているプロパティを差さなくなるので用途によっては賛否ありそうですが……。
```suggestion
                            missingProperties.Add(prop);
```
![image](https://user-images.githubusercontent.com/7193342/60677711-39f6e480-9ebd-11e9-8da3-ccf425b4ca34.png)

![image](https://user-images.githubusercontent.com/7193342/60677773-6448a200-9ebd-11e9-87ae-b6c54c754993.png)

Reply:
なるほど、マテリアルの方はこの方が良いですね！
修正取り込みます！